backend/spec/features/admin/taxons_spec.rb

```diff
@@ -21,4 +21,30 @@
     click_button "Update"
     expect(page).to have_content("Taxon \"Shirt\" has been successfully updated!")
   end
+
+  it "admin should be able to remove a product from a taxon", type: :feature, js: true do
+    taxon_1 = create(:taxon, name: 'Clothing')
+    product = create(:product)
+    product.taxons << taxon_1
+
+    visit spree.admin_taxons_path
+    select_clothing_from_select2
+
+    find('.product').hover
+    find('.product .dropdown-toggle').click
+    # TODO: This should be a click_link "Delete from taxon"
+    page.execute_script("$('.js-delete-product').click()")
```

Comment:
Thanks for the feedback (also on the other lines).
I used selenium-webdriver during development and somehow i couldn't click the link with `click_link` then.. after a lot of testing I discovered poltergeist gave no problems! Didn't know there could be differences betweens those drivers. So it's fixed!